UI/Web/src/app/manga-reader/_components/manga-reader/manga-reader.component.html

```diff
@@ -193,7 +193,9 @@
                 <option value="full-height">{{t('height')}}</option>
                 <option value="full-width">{{t('width')}}</option>
                 <option value="original">{{t('original')}}</option>
+                <option value="manual">{{t('manual')}}</option>
               </select>
+              <input id="page-fitting-slider" type="range" min=0 max=100 class="form-range visually-hidden" formControlName="widthSlider">
```

Comment:
You're going to need to have a label that's invisible for this. You need "" around your values for attr="value".